tests/test_export_dos_labs.py

```diff
@@ -0,0 +1,30 @@
+from dmscripts.export_dos_labs import append_contact_information_to_services
+
+
+def test_append_contact_information_to_services():
+    records = [
+        {
+            'services': [{'serviceName': 'service1'}, {'serviceName': 'service2'}],
+            'supplier': {'contactInformation': [{'email': 'me@me.com', 'other': 'value'}]}
+        },
+        {
+            'services': [{'serviceName': 'service3'}],
+            'supplier': {'contactInformation': [{'email': 'me2@me2.com', 'other': 'value'}]}
+        },
+    ]
+
+    res = append_contact_information_to_services(records, ['email'])
```

Comment:
It might be nice to test with all three of the same keys we use in the script itself?  Or at least more than one key, so we are sure it isn't getting things mixed up.

Reply:
Will add another key